src/rust/ide/enso-protocol/src/common.rs

```diff
@@ -0,0 +1,196 @@
+//! Commons of JSON-RPC-based Enso services.
+
+/// Time in UTC time zone represented as ISO-8601 string.
+pub type UTCDateTime = chrono::DateTime<chrono::FixedOffset>;
```

Comment:
It is not represented as a string, if you use `DateTime`. Though likely it is serialized as it.

Reply:
Sorry, this was copied from https://github.com/luna/enso/blob/master/doc/language-server/specification/enso-protocol.md#utcdatetime